### ISSUE TITLE
change avatar

### ISSUE THREAD

**Issue by moaeddy** [alievk/avatarify-python#437]
how do i change avatar when using python version?

and how do i fix this error

warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")

**Comment by JohanAR** [comment]
Sounds like there is something wrong with one of the files you put in the images folder. Try figuring out which file is causing the problem and delete it

**Comment by BILLO116** [comment]
How do I change my image in the folder because I don't know how to trace my img folder

**Comment by JohanAR** [comment]
Closing this issue since original creator appears to have solved their problems.

@BILLO116 it's the folder named "avatars". Check the documentation when in doubt :)
